doc/getting_started/installation_mac.rst

```diff
@@ -108,114 +108,21 @@ To build for the ARM-based Nordic nRF52 Development Kit:
 Setting Up the Toolchain
 ************************
 
-Install tools needed for building the toolchain (if needed):
+In case a toolchain is not available for the board you are using, you can build
+a the toolchain from scratch using crosstool-NG. Follow the steps on the
+crosstool-NG website to prepare your `host
```

Comment:
as written, only the word ``host`` would be clickable... better would be include ``prepare your`` too:

    website to `prepare your host
    <http://crosstool-ng.github.io/docs/os-setup/>`_